### PR TITLE
Fix `get_committed_offset` return specs

### PR DESCRIPTION
**Pull request by SerikDM**
Fix specs according to expected format in: https://github.com/kafka4beam/brod/blob/f88130e051fc2e317e744217f9e05071378d4629/src/brod_group_subscriber_v2.erl#L281

### CODE REVIEW

**Comment by zmstone** [comment]
thanks could you help to update Changelog.md ?

**Comment by SerikDM** [comment]
> thanks could you help to update Changelog.md ?

Sure, added a line about fixing specs

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+* 3.16.5
+  * Fix specs for `get_committed_offset`
```
Comment (zmstone) [inline changelog.md]: Please add module name too.

Comment (SerikDM) [reply changelog.md]: Added

**Pull request merged**